src/apps/testapps/testPentagonIndexes.c

```diff
@@ -56,6 +56,16 @@ SUITE(getPentagons) {
         }
     }
 
+    TEST(getPentagonsInvalid) {
+        H3Index h3Indexes[PADDED_COUNT] = {0};
+        t_assert(H3_EXPORT(getPentagons)(16, h3Indexes) == E_RES_DOMAIN,
+                 "getPentagons of invalid resolutions fails");
+        t_assert(H3_EXPORT(getPentagons)(100, h3Indexes) == E_RES_DOMAIN,
```

Comment:
For my info, is there any difference between this test and the one above? Having two tests implies two cases, but I think there's only one here.

Reply:
They should be the same code path, just different order of magnitude of the resolution parameter.

Reply:
Well, we do bit-packing in this library and many values are constrained well below an integer, so I can see a difference between a 5-bit value (the 16) and a 7-bit value (the 100) for our testing. If we simply lopped off the high bits they'd be turned into res 0 and res 4, respectively.